packages/mux-video/package.json

```diff
@@ -60,33 +92,35 @@
     "lint": "ESLINT_USE_FLAT_CONFIG=false eslint src/ --ext .js,.jsx,.ts,.tsx",
     "test": "web-test-runner **/*test.js --port 8002 --coverage --config test/web-test-runner.config.mjs --root-dir ../..",
     "posttest": "replace 'SF:src/' 'SF:packages/mux-video/src/' coverage/lcov.info --silent",
-    "dev:iife": "npm run build:iife -- --watch=forever",
```

Comment:
question(blocking): why were these dev scripts removed? feels out of scope.

Reply:
cool, I'll add the dev scripts back in.